src/core.c/Real.pm6

```diff
@@ -159,6 +159,7 @@ multi sub prefix:<->(Real:D $a) { -$a.Bridge }
 # so fails on differing types."  Thus no casts here.
 proto sub infix:<mod>($, $, *%) is pure {*}
 multi sub infix:<mod>(Real:D $a, Real:D $b) { $a - ($a div $b) * $b }
+multi sub infix:<mod>($a, $b) { $a.Real - ($a.Real div $b.Real) * $b.Real }
```

Comment:
Why not `(Real() $a, Real() $b)` as signature?

Reply:
Coercion signatures are quite a bit slower. E.g., `multi sub foo(Int $a, Int $b) { $a + $b - 2 }; multi sub foo($a, $b) { $a.Int + $b.Int - 2 }; my $a; $a = foo(~$_, $_ + 1) for ^1_000_000; say now - INIT now; say $a` reports ~0.5s, but `multi sub foo(Int() $a, Int() $b) { $a + $b - 2 }; my $a; $a = foo(~$_, $_ + 1) for ^1_000_000; say now - INIT now; say $a` reports ~1.0s.

Reply:
Ooh... in that case, that feels like low-hanging fruit ?